src/os/CMakeLists.txt

```diff
@@ -115,16 +120,22 @@ endif()
 
 if(WITH_SPDK)
   include_directories("${CMAKE_SOURCE_DIR}/src/spdk/include")
+  add_custom_target(build_dpdk
```

Comment:
@optimistyzy i have no preference either way. just wanted to put the pros and cons on the table to ensure that we are on the same page.